<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="GOT500VI73546" name="com.volvo.mfg.test.TestLoginPages" tests="6" failures="0" timestamp="26 Jul 2018 11:14:43 GMT" time="0.005" errors="2">
  <testcase name="MfgLogin" time="0.000" classname="com.volvo.mfg.test.TestLoginPages">
    <skipped/>
  </testcase> <!-- MfgLogin -->
  <testcase name="MfgLogin" time="0.000" classname="com.volvo.mfg.test.TestLoginPages">
    <skipped/>
  </testcase> <!-- MfgLogin -->
  <testcase name="MfgLogin" time="0.000" classname="com.volvo.mfg.test.TestLoginPages">
    <skipped/>
  </testcase> <!-- MfgLogin -->
  <testcase name="InitTestMethod" time="0.003" classname="com.volvo.mfg.test.TestLoginPages">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.volvo.ncl.reports.Reports.startTestCase(Reports.java:36)
at com.volvo.mfg.test.TestBase.InitTestMethod(TestBase.java:130)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:520)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- InitTestMethod -->
  <testcase name="MfgLogin" time="0.000" classname="com.volvo.mfg.test.TestLoginPages">
    <skipped/>
  </testcase> <!-- MfgLogin -->
  <testcase name="InitTestMethod" time="0.002" classname="com.volvo.mfg.test.TestLoginPages">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.volvo.ncl.reports.Reports.startTestCase(Reports.java:36)
at com.volvo.mfg.test.TestBase.InitTestMethod(TestBase.java:130)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:520)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- InitTestMethod -->
</testsuite> <!-- com.volvo.mfg.test.TestLoginPages -->
